core/utils/poly-look/src/visualisations/charts/bubble-charts/iconCluster/iconCluster.js

```diff
@@ -0,0 +1,84 @@
+import { BubbleCluster } from "../..";
+
+const bubblePadding = 15;
+
+/**
+ * Visualizes data as a cluster of icon (unicode-emoji or svg) where the value of the icon is represented as the size.
+ *
+ * The icons are being added in a spiral starting in the center of the cluster meaning sorted data will lead to all small icons in the middle or outside.
+ *
+ * @class
+ * @extends BubbleCluster
+ * @param {CSS-selector} selector - A CSS selector, where the svg will be attached to
+ * @param {Object[]} data - The data to be visualized as a bubble cluster
+ * @param {string} data[].title - A title for the icon node
+ * @param {number} data[].value - The value of the bubble, which corresponds to it's font-size/ svg-size
+ * @param {number} data[].icon - A svg that should be displayed as the bubble in string form as you would get it from rollup-svg-plugin (eg. "<svg>...</svg>")
+ * @param {string = null} data[].background - Color of a circle that's drawn circle is drawn and the icon is put inside
+ * @param {number = 400} width - The width of the svg
+ * @param {number = 300} height - The height of the svg
+ * @param {boolean = true} [showValues] - Whether texts displaying the value of the bubble are added
+ * @param {callback = () => {}} [onIconClick] - Icon onclick function
+ * @param {number|callback = 1} [opacity] - The opacity of the icons 0 <= opacity <= 1 (callbacks receive event and data)
+ * @param {number|callback = ""} [filter] - A filter that is applied to the icons eg saturate(0.5)(callbacks receive event and data)
+ */
+export class IconCluster extends BubbleCluster {
```

Comment:
I think it'd be nicer if we could get `BubbleCluster` to support this use case directly, but we can worry about that later.

Reply:
Just based on the input I'd say.

Reply:
I implemented it checks for data.icon